.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 💬 Question
```

Comment:
@ulfgebhardt I would not like to encourage people to submit questions via issues. For that we have Discord: https://discordapp.com/invite/6ub73U3

Reply:
I disagree - discord is chaotic and for chatting - while a question raised here has a more permanent nature. Therefore I suggest to implement it. A valid strategy to deal with questions would be
a: link to a part of the documentation
b: create documentation and then link

Reply:
On a side note: The Discord invite is stated in the template to let people think about hooking up there to get their question answered.

Reply:
If you disagree, I'm fine with it. Let's merge this and observe how many questions we get through the template. If we are unhappy with the outcome (stale issues, unclear definition-of-done) we can remove that issue template any time.

Reply:
I like the points that @ulfgebhardt brought up, I am a little nervous it could get out of hand as @roschaefer maybe is as well, but I think it's a nice compromise to deal with the issue if and when it comes up, not before